fe/fe-core/src/main/java/com/starrocks/credential/CloudConfigurationFactory.java

```diff
@@ -83,10 +84,10 @@ public static AWSCloudCredential buildGlueCloudCredential(HiveConf hiveConf) {
 
     public static CloudConfiguration buildCloudConfigurationForTabular(Map<String, String> properties) {
         Map<String, String> copiedProperties = new HashMap<>();
-        String sessionAk = properties.getOrDefault(AwsProperties.S3FILEIO_ACCESS_KEY_ID, null);
-        String sessionSk = properties.getOrDefault(AwsProperties.S3FILEIO_SECRET_ACCESS_KEY, null);
-        String sessionToken = properties.getOrDefault(AwsProperties.S3FILEIO_SESSION_TOKEN, null);
-        String region = properties.getOrDefault(AwsProperties.CLIENT_REGION, null);
+        String sessionAk = properties.getOrDefault(S3FileIOProperties.ACCESS_KEY_ID, null);
```

Comment:
Can you make sure that key is the same as before.

Reply:
Yes